src/actionItems.ts

```diff
@@ -1,3 +1,181 @@
+/**
+ * The type of actionItem a person will be notified to fulfill, without the assessments v1 action items
+ */
+// TODO: https://transcend.height.app/T-34487 - remove after monorepo FE and BE have migrated
+export enum ActionItemCodeWithoutAssessmentsV1 {
```

Comment:
nit: Can we rename the old one to say ActionItemCodeWithDeprecatedEntry and that way the "new" updated enum can have the same name as before?

Reply:
But won't that break the way the import works, as if I had just removed the values from `ActionItemCode` right now?

Reply:
@iamtheluckyest when you import the package into the monorepo, you'd update references to the backend to use the enum `ActionItemCodeWithDeprecatedEntry` so it wouldn't break i think?

Reply:
It still needs approval to merge though.

Reply:
It says I dismissed your review but I didn't! 😳 
<img width="869" alt="image" src="https://github.com/transcend-io/privacy-types/assets/19640576/9e167ae5-1c7b-40e2-be87-c4964e235ef7">

Can you approve again, plz? 🥺